### DISCUSSION TITLE
[BUG] faucet.chia.net

### DISCUSSION THREAD

**Discussion by chrismikefowler1** [Chia-Network/chia-blockchain#5475] [Support]
I use my receive address, and it keeps saying already received.

![image](https://user-images.githubusercontent.com/84326972/118680646-c4941480-b7b3-11eb-8a44-61adff215b74.png)

**Comment by sargonas** [comment]
Have you tried looking your wallet address up on Chia Explorer to see if the transaction is logged? It's possible it went through, but your local wallet is somehow out of sync and therefore not reflected in your GUI.

**Reply by horvathzeros** [reply]
Same happens to me, wallet sync ok, faucet "already sent" but nothing received. 

And nothing on chia explorer

**Comment by wwwean** [comment]
Same problem. Create new wallet and ... same problem. Perfect :)

**Reply by loppefaaret** [reply]
if our official faucet isn't working for you, there are some community faucets out there that works as well

**Comment by BrandtH22** [comment]
Since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
